rfc/2019-12-9-pingcap-university-campus.md

```diff
@@ -0,0 +1,65 @@
+# Proposal: Talent Plan Courses
+- Author(s): [Ling Jin](https://github.com/JinLingChristoher)
+- Last updated: 2019-12-6
+- Discussion at:
+
+## Abstract
+
+There are plenty of people interested in TiDB but lack enough background knowledge, especially college students and engineers without backend development experience. The proposal aims at helping those people enter the field of TiDB by creating a series of courses, which is also called Talent Plan Courses. Participants should be able to pull requests to TiDB related projects after consumed the course materials.
+
+## Background
+
+Since the campus is always an important member of the TiDB Community, and we hope more young blood from universities or other fields can embrace the TiDB ecosystem, so we would develop a distributed database system course to help people without enough background knowledge to enter the field. The courses will be consist of reading supplies and programming assignments. As there are already many accessible materials online, we can develop the content of the course by learning from them. We think the voice from the different populations is important for the success of the goal, especially college students, so we hope they can join us to make this happen.
```

Comment:
Consider combining these two sections. This is not a thesis paper, no abstract is needed. Keep it simple and short. Don't have to repeat the same thing twice.